src/Goblin.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "Enemy.hpp"
+
+class Goblin : public Enemy {
+private:
+	const static int health;
```

Comment:
These will probably have to be not consts eventually as they will change over time

Reply:
This is a `static` health which means the initial health. To get the actual health for this enemy, use `this->getHealth()`

Reply:
ah right so this is like MaxHealth and enemy.health is current health. looks good for now but a bit confusing since same name. Also we will have to change it to make enemies gain health over time but should be good for now